world_universities_and_domains.json

```diff
@@ -131197,5 +131190,389 @@
+      "ssmiet.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://iitj.ac.in/"
+    ],
+    "name": "Indian Institute Technology(IIt) Jodhpur",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "iitj.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://vcet.edu.in/"
+    ],
+    "name": "Vidyavardhini's College of Engineering and Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "vcet.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.ipchile.cl/"
+    ],
+    "name": "Ipchile, Instituto Professional de Chile",
+    "alpha_two_code": "CL",
+    "state-province": null,
+    "domains": [
+      "edu.ipchile.cl"
+    ],
+    "country": "Chile"
+  },
+  {
+    "web_pages": [
+      "https://www.ltu.se/"
+    ],
+    "name": "Lulea University of Technology",
+    "alpha_two_code": "SE",
+    "state-province": null,
+    "domains": [
+      "ltu.se"
+    ],
+    "country": "Sweden"
+  },
+  {
+    "web_pages": [
+      "https://www.uef.fi/en"
+    ],
+    "name": "The University of Eastern Finland (UEF)",
+    "alpha_two_code": "FI",
+    "state-province": null,
+    "domains": [
+      "uef.fi"
+    ],
+    "country": "Finland"
+  },
+  {
+    "web_pages": [
+      "https://www.42heilbronn.de/en/"
+    ],
+    "name": "42Heilbronn",
+    "alpha_two_code": "DE",
+    "state-province": null,
+    "domains": [
+      "student.42heilbronn.de"
+    ],
+    "country": "Germany"
+  },
+  {
+    "web_pages": [
+      "https://kedge.edu/"
+    ],
+    "name": "Kedge Business School",
+    "alpha_two_code": "FR",
+    "state-province": null,
+    "domains": [
+      "kedgebs.com"
+    ],
+    "country": "France"
+  },
+  {
+    "web_pages": [
+      "https://eurecom.fr/"
+    ],
+    "name": "EURECOM",
+    "alpha_two_code": "FR",
+    "state-province": null,
+    "domains": [
+      "eurecom.fr"
+    ],
+    "country": "France"
+  },
+  {
+    "web_pages": [
+      "https://ciisa.cl/"
+    ],
+    "name": "Instituto Profesional Ciisa",
+    "alpha_two_code": "CL",
+    "state-province": null,
+    "domains": [
+      "ciisa.cl"
+    ],
+    "country": "Chile"
+  },
+  {
+    "web_pages": [
+      "https://www.alcestergs.co.uk/"
+    ],
+    "name": "Alcester Grammar School",
+    "alpha_two_code": "UK",
+    "state-province": null,
+    "domains": [
+      "alcestergs.com"
+    ],
+    "country": "United Kingdom"
+  },
+  {
+    "web_pages": [
+      "https://www.upq.mx/"
+    ],
+    "name": "Universidad Politecnica de Queretaro",
+    "alpha_two_code": "MX",
+    "state-province": null,
+    "domains": [
+      "upq.edu.mx"
+    ],
+    "country": "Mexico"
+  },
+  {
+    "web_pages": [
+      "https://www.juventus.ch"
+    ],
+    "name": "Juventus Technikerschule",
+    "alpha_two_code": "CH",
+    "state-province": null,
+    "domains": [
+      "juventus.schule"
```

Comment:
The domain here is quite strange, let's double check with Emilia

Reply:
![image](https://github.com/TheQtCompany/university-domains-list/assets/163825797/34deec1b-ca51-4b87-b55b-24fa63637de3)
Looks to be valid